crates/hotshot/src/lib.rs

```diff
@@ -254,28 +254,19 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> SystemContext<TYPES, I> {
     pub async fn timeout_view(
```

Comment:
Maybe outside this PR's scope, but do we ever use this function anymore?

Reply:
Good catch!, nope it's not used, I'll clean it up in a different PR